lib/clients/analytics.ts

```diff
@@ -1,18 +1,16 @@
 import * as Ingest from '@voiceflow/general-runtime/build/lib/clients/ingest-client';
 import { DataAPI, State } from '@voiceflow/general-runtime/build/runtime';
-import { Response } from 'ask-sdk-model';
 
 import log from '@/logger';
 import { Config } from '@/types';
 
 import { AlexaRuntimeRequest } from '../services/runtime/types';
-import { InteractBody, TurnBody } from './ingest-client';
+import { InteractionBody, Payload, TraceBody } from './ingest-client';
 import { AbstractClient } from './utils';
 
-type Payload = Response | AlexaRuntimeRequest | null;
-
+const isAlexaRuntimeRequest = (p: Payload): p is NonNullable<AlexaRuntimeRequest> => (p ? 'type' in p : false);
```

Comment:
```suggestion
const isAlexaRuntimeRequest = (p: Payload): p is NonNullable<AlexaRuntimeRequest> => p && 'type' in p;
```

Reply:
had to add `?? false` at the end because the type doesnt allow for null/undefined, but great suggestion!